### ISSUE TITLE
Remove AWS dependency

### ISSUE THREAD

**Issue by ivanjaros** [u2takey/ffmpeg-go#102]
There is no place for such thing in a library like this. It can easily be made into additional library or custom code. But it should not be a dependency.

**Comment by piyongcai** [comment]
I think so too.

**Comment by rawnly** [comment]
it' used here for the tests i believe https://github.com/u2takey/ffmpeg-go/blob/master/examples/example_test.go

**Comment by IncredibleLaser** [comment]
I investigated this issue because of the file size of the resulting go binary which pointed to aws being the main culprit.

Anyhow, I don't use AWS and I'm not sure why it would be incorporated at ffmpeg wrapper level, but in case someone is curious, for my particular case, the binary built solely with `go build` shrank from 14MB to 4MB.

**Comment by giraffesyo** [comment]
Agreed, I was surprised to see aws imported when looking at [ffmpeg.go](https://github.com/u2takey/ffmpeg-go/blob/master/ffmpeg.go#L11). Also this is same issue as https://github.com/u2takey/ffmpeg-go/issues/88
